packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -410,6 +422,7 @@ class _ScaffoldLayout extends MultiChildLayoutDelegate {
   final bool extendBody;
   final bool extendBodyBehindAppBar;
   final EdgeInsets minInsets;
+  final EdgeInsets minViewPadding;
```

Comment:
@Piinks Hey, I am currently working on another scaffold issue.
some tests block me. it seems to be relative to this change.
I am wondering why not add `minViewPadding` to `shouldRelayout(oldDelegate)`, it should re-layout if it changes, right?

If the above is right, this test('Floating Action Button bottom padding not consumed by viewInsets') will fail, could you take a look at this?

Additional, the test was added in https://github.com/flutter/flutter/pull/34298 by you too :)

Reply:
Hey @xu-baolin! Can you share the issue you are working on? It will help me better understand. It's been a while since I worked on this so I am trying to refresh my memory. 😅

Reply:
Looking at `Floating Action Button bottom padding not consumed by viewInsets`, it makes sense to me, but the issue you are working on may provide a reason otherwise.